RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate NotificationPreferencesView - Typescript

## Proposed changes
Evaluate NotificationPreferencesView.

## Issue(s)	


## How to test or reproduce
Open a chat, go to Actions -> Notifications:
* Change the notifications options and check if all options still works fine.

## Screenshots

## Types of changes




## Checklist



## Further comments
